packages/core/src/model.ts

```diff
@@ -80,11 +80,11 @@ export class ChartModel {
 						);
 					} else {
 						const [start, end] = axesOptions[axis].domain;
-
-						// Filter out data outside domain
+						// Filter out data outside domain if that datapoint is using that axis (has mapsTo property)
 						allData = allData.filter(
 							(datum) =>
-								datum[mapsTo] >= start && datum[mapsTo] <= end
+								!(mapsTo in datum) ||
```

Comment:
what's this do?

Reply:
your model might contain data for different axes (ex: dual axes line chart)
![image](https://user-images.githubusercontent.com/14351335/104510708-8befee80-55b9-11eb-9cdd-f83aec293bcd.png)

when the function going through **all** the axes and "discarding"/filtering out the points that are outside of the custom domain of each axis, this check makes sure that it isnt discarding datapoints that are irrelevant to the current axis.